pandas/tseries/tests/test_timeseries.py

```diff
@@ -4441,6 +4441,15 @@ def test_nat_operations(self):
         self.assertEqual(s.min(), exp)
         self.assertEqual(s.max(), exp)
 
+    def test_round_nat(self):
```

Comment:
no this *should* work, on a Timestamp (it will just return the same). so no need to check for an exception. Your test *should* fail w/o a fix though

```
In [1]: pd.Timestamp('NaT')
Out[1]: NaT

In [2]: pd.Timestamp('NaT').round('s')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-05da84ef741e> in <module>()
----> 1 pd.Timestamp('NaT').round('s')

AttributeError: 'NaTType' object has no attribute 'round'
```

and of course the fix is to define these!